org.sonarlint.eclipse.ui/plugin.xml

```diff
@@ -929,5 +929,14 @@
            class="org.sonarlint.eclipse.ui.internal.preferences.SonarLintPreferencesInitializer">
      </initializer>
   </extension>
-	
+
+  <extension
+        point="org.eclipse.e4.ui.css.swt.theme">
```

Comment:
Any constraint on the minimal version of Eclipse?

Reply:
AFAICT, the E4 CSS-based theming capabilities are GA since 4.2 (Juno); I traced back the earliest commits on this extension point to 2010, so I am rather confident about support in our targeted versions.

Reply:
Great! I wonder if we could also use the same technic to improve rendering of other colors :)